gateway-messages/src/sp_to_mgs.rs

```diff
@@ -471,6 +474,15 @@ pub enum SpError {
     ImageBoardUnknown,
     /// The new image has a `BORD` key that does not match the current image
     ImageBoardMismatch,
+    /// Received a `ResetComponentTrigger` request without first receiving a
+    /// `ResetComponentPrepare` request. This can be used to detect a successful
+    /// reset on the SP_ITSELF. Not used for the ROT where the SP observes
+    /// the RoT reset and can report success.
+    ResetComponentTriggerWithoutPrepare,
+    /// There will be policy violations for some requests:
+    ///   - No image in SlotId (what suitability checks are needed?)
+    ///   - Lower epoch than current in SlotId
+    SwitchDefaultImageError(u32),
```

Comment:
These should also be covered by `gateway_messages::tests::versioning::v2::sp_response()` (there's a loop inside it that covers all the `SpError` variants)